udaykalra/portfolio/target/portfolio-1/style.css

```diff
@@ -1,9 +1,141 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+:root {
+  --main-color: #fff;
+  --head-color: #000000;
+  --body-color: #121212;
+  --text-color: #e7e7e7;
+  --light-color: #333333;
+}
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+html,
+body {
+  margin: 0;
+  height: 100%;
+  background-color: var(--body-color);
+}
+body {
+  background: 100%;
+}
+
+div.main-content {
+  background-color: var(--body-color);
+  color: var(--main-color);
+  font-family: 'Roboto', sans-serif;
+  height: 100%;
+}
+
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url('hero.jpg');
+
+  height: 50%;
+
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-text {
+  font-family: 'Roboto', sans-serif;
+  font-size: 40px;
```

Comment:
The style guide prefers shorthand properties:
https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties

It looks like one is available for font:
https://www.w3schools.com/css/css_font_shorthand.asp

Reply:
This has been implemented.